src/app/pages/plugins/plugins-installed/plugins-installed.component.ts

```diff
@@ -154,6 +155,7 @@ export class PluginsInstalledListComponent {
   public selectedPool;
   public activatedPool: any;
   public availablePools: any;
+  protected publicIp = '';
```

Comment:
This isn't necessarily a public IP, it should probably just read as privateIP

Reply:
but it is a public IP, it grabs it using a tool that queries whatismyip.com

Reply:
Oops nevermind me, crossed this up with jail IP and asigra